src/client/components/multi-form-editor/landing-page.js

```diff
@@ -5,32 +5,26 @@ const React = require('react');
 class LandingPage extends React.Component {
   render(){
     return h('div.landing-page', [
-      // main part of landing-page page
```

Comment:
Whereas I get the point of doing this, note that this does effectively block https://github.com/PathwayCommons/factoid/pull/184 due to conflicts.